internal/functions/coviddata/download-covid-data-total.go

```diff
@@ -22,9 +22,22 @@ import (
 )
 
 type downloadRequest struct {
-	Modified string       `json:"modified"`
-	Source   string       `json:"source"`
-	Data     []TotalsData `json:"data"`
+	Modified string    `json:"modified"`
+	Source   string    `json:"source"`
+	Data     []RawData `json:"data"`
```

Comment:
Shouldn't the `TotalsData` struct be removed too? Or is it used somewhere else than just here?